spiceaidocs/docs/components/data-connectors/debezium.md

```diff
@@ -15,26 +15,44 @@ datasets:
       debezium_transport: kafka # Optional. Only `kafka` is currently supported.
       debezium_message_format: json # Optional. Only `json` is currently supported.
       kafka_bootstrap_servers: broker1:9092,broker2:9092,broker3:9092 # Required. A comma separated list of Kafka broker servers.
+      kafka_security_protocol: SASL_SSL # Default is `SASL_SSL`. Valid values are `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT`, `SASL_SSL`.
+      kafka_sasl_mechanism: SCRAM-SHA-512 # Default is `SCRAM-SHA-512`. Valid values are `PLAIN`, `SCRAM-SHA-256`, `SCRAM-SHA-512`.
+      kafka_sasl_username: kafka # Required if `kafka_security_protocol` is `SASL_PLAINTEXT` or `SASL_SSL`.
+      kafka_sasl_password: ${secrets:kafka_sasl_password} # Required if `kafka_security_protocol` is `SASL_PLAINTEXT` or `SASL_SSL`.
+      kafka_ssl_ca_location: ./certs/kafka_ca_cert.pem # Optional. Used to verify the SSL/TLS certificate of the Kafka broker.
+      kafka_enable_ssl_certificate_verification: true # Default is `true`. Set to `false` to disable SSL/TLS certificate verification.
+
     acceleration:
       enabled: true # Acceleration is required for the debezium connector.
       engine: duckdb # `duckdb`, `sqlite` and `postgres` are supported acceleration engines for Debezium.
       refresh_mode: changes # Optional. If specified, this is required to be set to `changes` - any other value is an error.
       mode: file # Persistence is recommended to not have to rebuild the table each time Spice starts.
 ```
 
-:::warning[Limitations]
-Currently, only the `PLAINTEXT` protocol is supported for connecting to Kafka. SSL and SASL are not yet supported.
-:::
-
 ## Configuration
 
 ### Parameters
 
+`SASL` = Simple Authentication and Security Layer.
```

Comment:
> `SASL` =
This feels a bit awkward to have an equals sign in documentation that's not part of the code. It might be easier to include the definition the first time that `SASL` is used down below, ie:

> - `SASL_PLAINTEXT`: Plaintext communication; SASL (Simple Authentication and Security Layer) authentication.

Reply:
Updated